pkg/alerter/alerter.go

```diff
@@ -0,0 +1,245 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alerter
```

Comment:
The alerter is an emailer, would it make sense to rename it emailer?

Reply:
This code doesn't do any emailing. An "emailer" would probably be an appropriate Klaxon, though.